src/plugins/search/search.js

```diff
@@ -236,8 +236,13 @@ export function init(config, vm) {
         namespaceSuffix = matches[0];
       }
     }
-    paths.unshift(namespaceSuffix + '/');
-  } else {
+    if (
+      paths.indexOf(namespaceSuffix + '/') === -1 &&
+      paths.indexOf(namespaceSuffix + '/README') === -1
+    ) {
+      paths.unshift(namespaceSuffix + '/');
+    }
+  } else if (paths.indexOf('/') === -1 && paths.indexOf('/README') === -1) {
     paths.unshift('/');
   }
```

Comment:
One suggestion is to convert these if/else conditions to `isXXX` variables so the code is easier to read.

For example, instead of this:

```js
if (
  paths.indexOf(namespaceSuffix + '/') === -1 &&
  paths.indexOf(namespaceSuffix + '/README') === -1
) {
 // ...
}
```

Consider:

```js
const isCondition1 = paths.indexOf(namespaceSuffix + '/') === -1;
const isCondition2 = paths.indexOf(namespaceSuffix + '/README') === -1;

if (isCondition1 && isCondition1) {
  // ...
)
```

Reply:
done.